### PR TITLE
M1 compatibility note + prettier

### PR DESCRIPTION
**Pull request by bigtedde**
Changes:
- Added M1/M2 compatibility note
- Ran prettier
- Fixed minor typos
- Corrected formatting on broken hyperlink

### CODE REVIEW

File: install/windows.md [hunk 1]
```diff
@@ -23,82 +24,95 @@ First enable Windows Subsystem for Linux
...
 After that you can run Vorta with
+
 ```
 $ vorta
 ```
 
 ## Possible Errors
 
-- If you get error, `ImportError: libQt5Core.so.5 :  cannot open shared object file: No such file or directory`, run
+- If you get error, `ImportError: libQt5Core.so.5 : cannot open shared object file: No such file or directory`, run
```
Comment (Hofer-Julian) [inline install/windows.md:82]:
```suggestion
- If you get an error, `ImportError: libQt5Core.so.5 : cannot open shared object file: No such file or directory`, run
```

**Review by Hofer-Julian — APPROVED** [review]
Thanks!

**Review by m3nu — APPROVED** [review]

File: install/windows.md (hunk 1 above)
Comment (bigtedde) [reply install/windows.md]: Done

**Comment by Hofer-Julian** [comment]
Thanks!

**Pull request merged**